dataactcore/models/views.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy import MetaData, Table
+from dataactcore.interfaces.db import GlobalDB
+
+
+class SubmissionUpdatedView:
```

Comment:
Not that I'm aware of, I just grabbed other classes and copied them (the models I figured were very slightly different because they're used by alembic directly). Do you have a preference?